master/cmd/determined-master/root.go

```diff
@@ -55,6 +56,12 @@ func runRoot() error {
 	}
 	log.Infof("master configuration: %s", printableConfig)
 
+	err = os.MkdirAll(config.Cache.CacheDir, fs.ModePerm)
+	if err != nil {
+		log.WithError(err).Errorf("Failed allocating cache direatory at %s", config.Cache.CacheDir)
```

Comment:
Nit: it is not called "allocating" a directory.  How about: `failed to make cache directory (%s)`?